java/client/test/org/openqa/selenium/build/BazelBuild.java

```diff
@@ -40,7 +40,7 @@ public void build(String target) {
     if (target == null || "".equals(target)) {
       throw new IllegalStateException("No targets specified");
     }
-    log.info("\nBuilding " + target + " ...");
+    log.finest("\nBuilding " + target + " ...");
```

Comment:
This is to let people know that the tooling is doing something during a build. Please leave.